app/interface/api.ts

```diff
@@ -18,6 +18,37 @@ export class Api {
         },
     ) {}
 
+    sendSummaryFeedbackHandler: Handler = async (params) => {
+        try {
+            const request = new Request(params);
+            var { to, context }: MailSummaryFeedback = {
+                to: request.parseString('to'),
+                context: {
+                    nombre: request.parseString('student.full_name'),
+                    ejercicios: request.parseString('summary.ejercicios'), // TODO: parsear a array
+                    nota_cursada_final: request.parseString('summary.grade_final_completed'),
+                    promedio_ejercicios: request.parseString('summary.prom_ej'),
+                    promedio_ej_primer_parcial: request.parseString('summary.prom_ej_1p'),
+                    primer_parcial: request.parseString('summary.fist_parcial'),
+                    segundo_parcial: request.parseString('summary.second_parcial_papers'),
+                    primer_recu: request.parseString('summary.first_recu_papers'),
+                    condicion_final: request.parseString('summary.final_condition'),
+                    punto_adicional: request.parseString('summary.extra_point'),
+                    nota_cursada: request.parseString('summary.grade_completed'),
+                    segundo_recu: request.parseString('summary.second_recu'),
+                },
+            };
+        } catch (error) {
+            return Response.badRequest(String(error));
+        }
+        try {
+            await this._services.mailer.sendSummaryFeedback(context, to);
+        } catch (error) {
+            return Response.error(String(error));
+        }
+        return Response.ok(`Correo enviado a ${to}`);
```

Comment:
El MailerClient no está configurado porque al final se mandan por el script de Google. Haria que devuelva las opciones y se siga desde el script.

```suggestion
        try {
            return Response.ok(this._services.mailer.sendSummaryFeedback(context, to));
        } catch (error) {
            return Response.error(String(error));
        }
```